server/src/handlers/Movies/moviesHandler.js

```diff
@@ -1,5 +1,7 @@
 const sqlQuery = require("../../database/my_sql_query");
 const dbConnection = require("../../database/db_connection");
+const { get } = require("../../routes/usersRoutes");
+const e = require("express");
```

Comment:
Can you explain what is the purpose of these lines? @Bill-Fashion

Reply:
I'm not the one who wrote these lines

Reply:
![image](https://user-images.githubusercontent.com/66483590/125105536-c2da3380-e108-11eb-96c6-5d2948dd91b9.png)
Are you sure? Git detected you add 2 code lines with the older version! @Bill-Fashion

Reply:
Alot of sure, the things i've done is that add code below the Việt's code and didn't care about the require

Reply:
Please remove that if we don't use that for any reason @Bill-Fashion

Reply:
maybe Việt wrote that. I've changed the name of the route, added auth to the get movie, and pushed the change to git.

Reply:
![image](https://user-images.githubusercontent.com/66483590/125106794-43e5fa80-e10a-11eb-9316-518078ea3424.png)

I pretty sure Viet didn't do that because he know those codes aren't necessary. Then, you push those codes from your laptop and your git, show I checked the last time I pushed the new one I didn't see those codes in our history. Now your code is the newest code in branch staging, so you wrote that. So, please remove the redundant codes and push it again @Bill-Fashion

Reply:
I think about the situation you pull that from gitHub then merge to you laptop. But, git only detects the changes which haven't had before. So, those codes are yours and please becareful ! @Bill-Fashion

Reply:
I know git detected code from mine but for any reason, i'm pretty sure that i'm not the one who wrote that code. But its ok, tomorrow i will remove it.

Reply:
ok, let's remove that asap to avoid the conflicts from your partner @Bill-Fashion

Reply:
Space between {} and get is not my style.